api/v1beta1/flowcollector_types.go

```diff
@@ -220,6 +220,20 @@ type FlowCollectorEBPF struct {
 	// (for example GOGC, GOMAXPROCS env vars). Users setting its values do it at their own risk.
 	// +optional
 	Debug DebugConfig `json:"debug,omitempty"`
+
+	// enable TCP drop flows logging feature, this feature requires mounting
+	// kernel debug filesystem hence the ebpf pod has to run as privileged
+	// if privileged isn't set an error will be reported.
```

Comment:
Some doc format:

```suggestion
	// Enable TCP drop flows logging feature. This feature requires mounting
	// kernel debug filesystem hence the eBPF pod has to run as privileged.
	// If privileged is not set, an error will be reported.
```

cc @skrthomas : any other suggestions?

Reply:
I made another suggestion. One question `privileged` here is the `spec.agent.eBPF.privileged` parameter? If so, I suggest:
> If the `spec.agent.eBPF.privileged` parameter is not set, an error is reported.